modules/rostests/apitests/shell32/GetDisplayNameOf.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:         ReactOS API tests
```

Comment:
```suggestion
 * PROJECT:     ReactOS API tests
```
Copyright:And_then_Three_Spaces is what we do recommend here:
https://reactos.org/wiki/Coding_Style#File_Structure
Sure, that is obscure nitpicking, I just noticed the difference here.